### DISCUSSION TITLE
How to re-download - torrent delete&added from/to the tracker

### DISCUSSION THREAD

**Discussion by ostracizado** [Flexget/Flexget#3850] [Q&A] [answered]
It's a problem I've had once or twice.

Ex: There's a torrent that's added to a tracker (rss - new) / the download starts and was finished.
But latter on the torrent it's delete because there was some problem with it or the file...

Even if I delete the torrent from deluge and from the download folder, flexget doesn't re-download the new torrent when it's added to the tracker and the rss list.

Is there a specific plugin I have to reset?

**Comment by crawln45** [comment, marked as answer]
There is actually a setting called propers: yes which will look for replaced torrents. Usually they say repacked, proper, ect. And flexget will automatically pull them. Otherwise you can forget them from seen and the new file will usually be downloaded again
